locus/src/main/java/com/birjuvachhani/locus/LocusActivity.kt

```diff
@@ -66,10 +74,10 @@ class LocusActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRe
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_location_permission)
-
+        config = Configuration(getLocusLocationRequest(getAvailableService()))
 
         intent?.getParcelableExtra<Configuration>(Constants.INTENT_EXTRA_CONFIGURATION)?.let {
-            config = it
+            config = it.copy(locationRequest = getLocusLocationRequest(getAvailableService()))
```

Comment:
Why are you calling `copy` and resetting `LocationRequest` to default configuration?

Reply:
Done.